os/arch/arm/Kconfig

```diff
@@ -115,6 +128,10 @@ config ARMV7M_MPU_NREGIONS
 
 config ARCH_HAVE_LOWVECTORS
 	bool
+	default y
```

Comment:
We don't recommand this as a default because this depends on each board. And this is not our functionality.
Let's keep it original.

Reply:
it will be change as your guidance. 
BR, HJ Lee.